driver-async/src/main/com/mongodb/async/client/AggregateFluent.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.async.client;
+
+import com.mongodb.async.SingleResultCallback;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Fluent interface for aggregate.
+ *
+ * @param <T> The type of the result.
+ * @since 3.0
+ */
+public interface AggregateFluent<T> extends MongoIterable<T> {
```

Comment:
I really don't like including the "Fluent" suffix in all these interface names, now that I see all these new ones.  What do you think about changing the suffix to "Iterable", e.g.  AggregateIterable?